src/TupleTypes.jl

```diff
@@ -2,15 +2,18 @@ module TupleTypes
 
 import Base: isvatuple
 
-function parameters{T<:Tuple}(::Type{T})
```

Comment:
As far as I understand it, the whole `Vararg` design is not settled in Julia.  So maybe not worth spending too much time on this.  I changed it such that it is up to the user to deal with the `Vararg`.